cmd/sink/jsonPath/jsonPath.go

```diff
@@ -0,0 +1,74 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+package jsonpath
+
+import (
+	"fmt"
+	"regexp"
+
+	"k8s.io/client-go/util/jsonpath"
+)
+
+// Evaluates the JSON Path, path, against the json, data, using jsonpath from k8s.io/client-go/util and returns a bool
+// representing whether or not the JSON Path exists in data. It expects find only one result. If the evaluation fails,
+// it returns the error
+func PathExists(path string, data map[string]interface{}) (bool, error) {
+	jp := jsonpath.New("Parser")
+	err := jp.Parse(path)
+	if err != nil {
+		return false, err
+	}
+	results, err := jp.FindResults(data)
+	if err != nil {
+		return false, err
+	}
+	if len(results) != 1 || len(results[0]) != 1 {
+		return false, fmt.Errorf("Expected 1 result from JSON Path")
```

Comment:
You should include what you were trying to do, or the error will not make much sense to the person debugging it.

```suggestion
		return false, fmt.Errorf("Failed to find a match, expected one result from JSON Path")
```

Reply:
done